### ISSUE TITLE
Just running 'vv' for the first time will prompt for vv path, but not save it

### ISSUE THREAD

**Issue by bradp** [bradp/vv#177]

**Comment by XedinUnknown** [comment]
I'm under Cygwin on Win10. Running anything prompts for password every time.
```
Xedin Unknown ~
$ vv list
[Error] Path specified is not a VVV root directory. Where is VVV installed?
VVV install directory: ~/projects/XedinUnknown/vvv-local/
        * wpra-test           ( wpra-test.dev )

Xedin Unknown ~
$ vv list
[Error] Path specified is not a VVV root directory. Where is VVV installed?
VVV install directory: ~/projects/XedinUnknown/vvv-local/
        * wpra-test           ( wpra-test.dev )

Xedin Unknown ~
$ vv list
[Error] Path specified is not a VVV root directory. Where is VVV installed?
VVV install directory:
```

**Comment by XedinUnknown** [comment]
Any news? Is this confirmed? Or maybe, I'm doing it wrong?

**Comment by XedinUnknown** [comment]
Hi,

Forget Cygwin. I installed Ubuntu on a laptop now. Still, every time I run anything `vv`, it says the following:
```
Automagically found  
 Is this where vv is installed? (Y/n):
```

My `--debug--vv`

```
VVV install directory: ~/archive/systems/vvv/
----------
vv version 1.9.3

vv: /home/xedinunknown/systems/vv/vv

vvv path: /home/xedinunknown/archive/systems/vvv
home: /home/xedinunknown

tput: /usr/bin/tput
cat: /bin/cat
curl: /usr/bin/curl
brew: 
tar: /bin/tar
find: /usr/bin/find
git: /usr/bin/git
sed: /bin/sed
paste: /usr/bin/paste
vagrant: /usr/bin/vagrant
----------
```

**Comment by davidjray** [comment]
@XedinUnknown Did you install via Homebrew or just install the script (via Git or whatever)?

**Comment by XedinUnknown** [comment]
@davidjray, I cloned the Git repo.

**Comment by davidjray** [comment]
@XedinUnknown I had the same problem after installing directly via the GIT repo. I am not sure how to fix the script itself, but if you manually edit "~/.vv-config" file it should stop prompting.

**Comment by markidm** [comment]
Also have the same problem, @davidjray your solution didn't work for me...

**Comment by davidjray** [comment]
Next time you see the message "Is this where vv is installed? (Y/n):" Try replying with Y. You should later get a message similar to "Path specified is not a VVV root directory. Where is VVV installed?". Or not at all. I am trying to recall the specific steps. 

Another thing to make sure is that your path is setup correctly.

**Comment by XedinUnknown** [comment]
The message in itself is misleading, as it doesn't even specify the path, in which VV thinks that VVV might be.
Anyhow, on my previous attempt in Windows I had tried that, and it didn't work. Anyone?
And where's the repo owner??

**Comment by bradp** [comment]
This is a known issue and going to be fixed in the next version.

https://github.com/bradp/vv/issues/161
https://github.com/bradp/vv/issues/179

**Comment by davidjray** [comment]
@bradp Thanks! Good work on this.

@XedinUnknown @markidm make sure you have your path setup correctly. For me this didn't work:``` *VVVpath*/vv/vv``` but this did: ``` *VVVpath*/vv```.

My path output ```/devvv/wordpress-vvv/vv:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin``` after running ```echo $PATH```

**Comment by XedinUnknown** [comment]
I installed VV as a sibling to VVV, i.e. not in a VVV subdirectory.

**Comment by XedinUnknown** [comment]
I tried suggestion of @davidjray, and it turned out that I don't have the `~/.vv-config` file. I ran `vv` again, and this time chose "yes" when it asked me if the autodetected path is correct. It created the config file, but kept telling me that the path is not a VVV base path. Upon inspection of the config file, it turned out to be a JSON object with one member, `path`. The value was empty. I manually entered the correct value for it, and it worked.
